src/cmd/seal.sh

```diff
@@ -19,14 +19,9 @@ cmd_seal() {
         rm -f "$file.sealed"
     fi
 
-    # get recipients
     get_gpg_key
-    local recipients="$GPG_KEY"
-    while [[ -n "$1" ]]; do
-        recipients="$recipients $1"
-        shift
-    done
-
+    local recipients="$GPG_KEY $@"
+    
```

Comment:
Divesh, this seems simpler, but does not work correctly in all the cases. For example if one of the recipients is specified as `'Firstname Lastname'`, this approach will convert it into 2 recipients: `Firstname` and `Lastname`.
The correct approach is the one that I told you, using arrays and double quotes, although it looks more ugly.